verification/abi.go

```diff
@@ -77,6 +101,10 @@ type GetProfileResp struct {
 
 type CertifyKeyFlags uint32
 
+const (
+	CertifyAddIsCA CertifyKeyFlags = 1 << 30
```

Comment:
Is the `Add` part here a typo? I'm thinking this should either be `CertifyKeyIsCA` or simply `IsCA`.

Reply:
Technically the flag in the ABI is `ADD_IS_CA` which is short for "Additional Input: IS_CA". This is because the mainline DPE spec defines a parameter "Additional Input", which is additional data used to populate the cert. And then it is up to the profile to decide what additional input is supported.

But this is probably minutiae the caller doesn't care about, so I'll remove the add from here.

Reply:
Oh, I missed that in the ABI. You can leave it as is.